notify-slack-deploy/notify.py

```diff
@@ -27,8 +28,18 @@
     "value": f"{actor} {run_description} <{html_url}/actions/runs/{run_id}|{workflow}> {run_emoji}",
     "short": False,
 }
+fields = [field]
+if os.environ["CUSTOM_MESSAGE"]:
+    fields.append({"value": os.environ["CUSTOM_MESSAGE"]})
+
 body = {
-    "attachments": [{"fallback": fallback, "color": color, "fields": [field]}],
+    "attachments": [{"fallback": fallback, "color": color, "fields": fields}],
 }
 
-print(requests.post(os.environ["SLACK_WEBHOOK"], json=body))
+request = urllib.request.Request(
+    os.environ["SLACK_WEBHOOK"], 
+    json.dumps(body).encode("utf-8"),
+    { "Content-Type": "application/json" }
+)
+
+urllib.request.urlopen(request)
```

Comment:
Can you print the output of this? Most people won't look at it, but I can imagine it's helpful for debugging.

Reply:
Just printing that object wasn't very enlightening, so I added some code to print the status and the response body. You can see the results [here](https://github.com/mbta/gtfs_creator/runs/2886617852?check_suite_focus=true).